internal/resources/cluster/nodepools/resource_node_pool.go

```diff
@@ -239,25 +463,6 @@ func flattenTkgServiceVsphere(tkgServiceVsphere *nodepoolmodel.VmwareTanzuManage
 	return []interface{}{flattenTkgServiceVsphereData}
 }
 
-func constructTkgServiceVsphere(data []interface{}) (tkgServiceVsphere *nodepoolmodel.VmwareTanzuManageV1alpha1ClusterNodepoolTKGServiceVsphereNodepool) {
```

Comment:
Why have we removed this construct function?

Reply:
It is present [here.](https://github.com/vmware/terraform-provider-tanzu-mission-control/blob/acb97de32f9c5bcbed468231c3458ac6f7509eed/internal/resources/cluster/nodepools/resource_node_pool.go#L352) or else our tests would have been failing 😅